libsignal-service/src/lib.rs

```diff
@@ -43,4 +43,5 @@ pub mod prelude {
         receiver::MessageReceiver,
         sender::{MessageSender, MessageSenderError},
     };
+    pub use phonenumber::{parse, PhoneNumber};
```

Comment:
```suggestion
    pub use phonenumber::{parse as parse_phonenumber, PhoneNumber};
```

Maybe? Because otherwise we're having a loose `parse()` function in our `prelude`.